src/config.template.cjs

```diff
@@ -142,14 +142,14 @@ module.exports = {
 	 * recommended to set this to false.
 	 * Only applies to data based matches.
 	 */
-	skipRecheck: false,
+	skipRecheck: true,
 
 	/**
 	 * Determines how deep into the specified dataDirs to go to generate new searchees.
 	 * Setting this to higher values will result in more searchees and more API hits to
 	 * your indexers.
 	 */
-	maxDataDepth: 2,
+	maxDataDepth: 1,
```

Comment:
I'm confused on why this is change is necessary, does this prevent diving into nested directories? I think if you have the clients data setup and any sort of sorting, it would prevent effective searching?

I think path depth is something I'm not entirely certain on, though, or the purpose or benefit in reducing this.

Reply:
Seeing as this is just OOTB defaults, I think I'd rather err on the side of less tracker spam here. 
> I think if you have the clients data setup and any sort of sorting, it would prevent effective searching?

Not sure what you mean by sorting, but if you mean putting `/downloads` as your data dir instead of `/downloads/movies` and `/downloads/shows`, that's never been officially supported and I think it makes bogus searches for "shows" and "movies". I think ideally we just ask people do do the latter. It will be fairly apparent what's wrong when people make that mistake as well.

Reply:
Maybe a note in the text above (and/or in the docs) mentioning that if someone is using the default for `maxDataDepth` they should have more explicit `dataDir` paths (`dataDir: ["/path/to/torrents/category"]` vs `dataDir: ["/path/to/torrents"]`).